ui/plugins/controller.js

```diff
@@ -329,9 +329,7 @@ treeherder.controller('PluginCtrl', [
             thBuildApi.cancelAllJobs($scope.repoName, rs.revision);
         };
 
-        /**
-         * Test whether or not the selected job is a reftest
-         */
+        // Test to expose the reftest button in the job details navbar
         $scope.isReftest = function() {
```

Comment:
> However for the job details above, the reftest icon flashes off/on as the evaluation happens through the provider.

Hah. Seems I can get better UX using `$scope.selectedJob` instead of `$scope.job`. So I think I can use the provider for both.